services/src/main/java/org/keycloak/broker/provider/ScriptMapper.java

```diff
@@ -0,0 +1,131 @@
+package org.keycloak.broker.provider;
+
+import org.jboss.logging.Logger;
+import org.keycloak.broker.saml.SAMLIdentityProviderFactory;
+import org.keycloak.models.*;
```

Comment:
Minor: Will be good to remove star imports and import all classes separately (See other files of Keycloak codebase).

Reply:
Done